### PR TITLE
Added home page in my portfolio project

### PR DESCRIPTION
**Pull request by neetu-prasad**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,17 +28,27 @@
...
+
+            <div class="singleline">
+
+                <p class="name-font">I'm <b>Neetu Uday Prasad </b></p>
+
+                <p class="about">Enterprneur, front-end developer, UI & UX designer and student.</p>
             </div>
-            <div>
-                image
+            <div class="image-one">
```
Comment (neetu-prasad) [inline index.html]: please give meaningful class name

File: index.html [hunk 2]
```diff
@@ -28,17 +28,27 @@
...
         </nav>
         <div class="content-container">
-            <div>
-                write some thing
+
+            <div class="singleline">
+
+                <p class="name-font">I'm <b>Neetu Uday Prasad </b></p>
+
+                <p class="about">Enterprneur, front-end developer, UI & UX designer and student.</p>
```
Comment (neetu-prasad) [inline index.html]: give class name as info

File: index.html [hunk 3]
```diff
@@ -28,17 +28,27 @@
             </div>
         </nav>
         <div class="content-container">
-            <div>
-                write some thing
+
+            <div class="singleline">
```
Comment (neetu-prasad) [inline index.html]: give some another class name

File: style.css [hunk 4]
```diff
@@ -26,6 +35,52 @@ html, body {
...
+    background-position: center;
+}
+
+.social-media-container i{
+    font-size: large;
+    padding: 10px;
+    
+}
+
+.email-container{
```
Comment (neetu-prasad) [inline style.css]: remove empty class

File: style.css [hunk 5]
```diff
@@ -26,6 +35,52 @@ html, body {
...
+    
+}
+
+body{
```
Comment (neetu-prasad) [inline style.css]: move this class at the top of file

File: style.css [hunk 6]
```diff
@@ -26,6 +35,52 @@ html, body {
...
+}
+
+.footer{
+    display: flex;
+    justify-content: space-between;
+    padding: 4px 80px;
+}
+
+.matter{
+    margin-right: 10px;
```
Comment (neetu-prasad) [inline style.css]: instead added right margin here. add left-margin in image div class

**Review by uday28kumar — APPROVED** [review]

**Pull request merged**